vhs-frontend/src/types/types.ts

```diff
@@ -19,40 +20,40 @@ export type VHS = Omit<VhsFormType, 'thumbnail'> & {
 };
 
 export const vhsFormSchema = z.object({
-  title: z.string().min(1, 'Title is required'),
-  description: z.string().min(1, 'Description is required'),
-  genre: z.string().min(1, 'Genre is required'),
+  title: z.string().min(1, `${i18n.t('validationErrors.title')}`),
```

Comment:
You can just put `i18n.t()` without template string quotes.

Reply:
If you're getting an error because the return type of i18n.t is not a string, hit me up on Discord.